src/view.js

```diff
@@ -302,8 +302,8 @@ export default class View {
 
     // Set general uniforms that must be updated once per program
     setupProgram (program) {
-        program.uniform('2f', 'u_resolution', this.size.device.width, this.size.device.height);
-        program.uniform('3f', 'u_map_position', this.center.meters.x, this.center.meters.y, this.zoom);
+        program.uniform('2fv', 'u_resolution', [this.size.device.width, this.size.device.height]);
```

Comment:
We still have unnecessary arrays being created in cases like this, but these aren't actually "new" arrays since the spread operator was creating them before anyway. But we can still factor them out later if it helps (not in the scope of this PR).